Description: Use libdl to build unittests

Author: Sudip Mukherjee <sudipm.mukherjee@gmail.com>
Forwarded: no

---

diff --git a/utest/Makefile b/utest/Makefile
index 2cf9974..daca400 100644
--- a/utest/Makefile
+++ b/utest/Makefile
@@ -10,7 +10,7 @@ OBJS =
 OBJS += trace-utest.o
 OBJS += tracefs-utest.o
 
-LIBS += -lcunit $(LIBTRACEEVENT_LDLAGS) $(LIBTRACEFS_LDLAGS)
+LIBS += -lcunit $(LIBTRACEEVENT_LDLAGS) $(LIBTRACEFS_LDLAGS) -ldl -ltracefs -ltraceevent
 
 OBJS := $(OBJS:%.o=$(bdir)/%.o)
 DEPS := $(OBJS:$(bdir)/%.o=$(bdir)/.%.d)
